apps/api/src/campaign/campaign.service.ts

```diff
@@ -73,6 +73,7 @@ export class CampaignService {
             person: { select: { id: true, firstName: true, lastName: true } },
           },
         },
+        campaignFiles: true,
```

Comment:
may be the campaign files are not needed to be returned each time, especially if the list is long. The suggestion is to load the file list in separate query where paging could be implemented too.

Reply:
fair point. I will add an endpoint to get the campaign files for a campaign

Reply:
This is actually the "campaign details" query. So this is only returned when you get a single campaign. Listing the campaigns does not include the campaign files.

Reply:
ok, let's leave it as is and we can optimize later.